src/desktop/components/stylus_lib/index.styl

```diff
@@ -19,7 +19,7 @@ gutter = 30px
 header-logo-icon-size = 40px
 home-logo-size = 60px
 layout-side-margin = 55px
-main-header-height = 61px
+main-header-height = 59px
```

Comment:
This all corresponded to old bar. I clicked around and everything seems ok in terms of old apps, but fixes gap that we see in new pages